FirebaseCore.podspec

```diff
@@ -53,6 +53,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   # Remember to also update version in `cmake/external/GoogleUtilities.cmake`
   s.dependency 'GoogleUtilities/Environment', '~> 7.6'
   s.dependency 'GoogleUtilities/Logger', '~> 7.6'
+  s.dependency 'GoogleUtilities/NSData+zlib', '~> 7.6'
```

Comment:
.

Reply:
Is there some other config I'm missing? @paulb777

Reply:
Maybe related: https://stackoverflow.com/questions/67894280/how-to-import-a-cocoapods-subspec-module-instead-of-the-entire-pod-in-swift